components/operator/garden/nats/garden.yaml

```diff
@@ -1,16 +1,17 @@
 ---
-apiVersion: garden.io/v0
```

Comment:
maybe keep the version field on v1?

Reply:
I've added back, the apiVersion description, to all Build, Deploy actions